content/en/service_management/incident_management/declare.md

```diff
@@ -43,6 +43,12 @@ You can declare an incident directly from a graph by clicking the export button
 
 {{< img src="service_management/incidents/from-a-graph.png" alt="Create in incident from a graph" style="width:80%;">}}
 
+## From a Synthetic test
+
+Create incidents directly from your [Synthetic tests page][8] from the Actions dropdown. Select **Declare incident** to open an incident creation modal, where a summary of the test is added to your incident timeline, allowing you to pursue the investigation from there.
+
+{{< img src="service_management/incidents/declare/synthetics_declare_incident.png" alt="Declare an incident from a Synthetic test." style="width:90%;" >}}
```

Comment:
This is a personal preference, but I like to show the entire width of a page so that a user can see where the button is, and on what page they should be. Since the link is for the list of Synthetics tests and not a specific synthetic test, it might be worth showing the full width of a synthetic test page.